integration/tests/cook/cli.py

```diff
@@ -90,20 +90,36 @@ def __show_json(uuids, cook_url=None, flags=None):
     return cp, data
 
 
-def show_json(uuids, cook_url=None, flags=None):
+def show_jobs(uuids, cook_url=None, flags=None):
     """Shows the job JSON corresponding to the given UUID(s)"""
     cp, data = __show_json(uuids, cook_url, flags)
     jobs = [job for entities in data['clusters'].values() for job in entities['jobs']]
     return cp, jobs
 
 
+def show_instances(uuids, cook_url=None, flags=None):
+    """Shows the instance JSON corresponding to the given UUID(s)"""
+    cp, data = __show_json(uuids, cook_url, flags)
+    instance_job_pairs = [pair for entities in data['clusters'].values() for pair in entities['instances']]
+    return cp, instance_job_pairs
```

Comment:
Again, are we changing the way instances are displayed? Assuming so, does this need to change?

Reply:
:ok: I think it makes sense to do this in a separate PR. I've created an issue to track this: #589.